src/network/sniffer.rs

```diff
@@ -1,12 +1,12 @@
 use ::std::boxed::Box;
 
-use ::pnet::datalink::{DataLinkReceiver, NetworkInterface};
-use ::pnet::packet::ethernet::{EtherType, EthernetPacket};
-use ::pnet::packet::ip::IpNextHeaderProtocol;
-use ::pnet::packet::ipv4::Ipv4Packet;
-use ::pnet::packet::tcp::TcpPacket;
-use ::pnet::packet::udp::UdpPacket;
-use ::pnet::packet::Packet;
+use ::pnet_bandwhich_fork::datalink::{DataLinkReceiver, NetworkInterface};
```

Comment:
We could avoid the renaming by leveraging [this](https://doc.rust-lang.org/1.31.0/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml) feature of cargo

Reply:
Huh - cool. I'm just wondering if this works for binary crates installed from cargo too? I know a lot of similar stuff (eg. patch) doesn't.

Reply:
Oh, did not know about those limitations. Better test it first, then.

Reply:
How? :) The only test I can think of is publishing a fake package that will be in crates.io forever (unless they have a <24h yank policy?)

Reply:
Damn, you are right. Alright then, lets play this safe :+1: